### ISSUE TITLE
[TC-MF-1.17] [TC-MF-1.18] - Removable of fabric that is when we send remove-fabric command in chip-tool of TH2 makes DUT abort.

### ISSUE THREAD

**Issue by sowmyassp** [project-chip/connectedhomeip#17162]
- Issue : 

[TC-MF-1.17] [TC-MF-1.18], Step 8 of test-plan for the mentioned test-cases says to remove fabric in TH2 which  makes DUT abort.

Test-plan reference : https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/multiplefabrics.adoc#tc-mf-1-17-removing-fabrics-from-dut-and-fabric-index-enumeration-using-ecm-dut-commissionee

Test platform :
Chip-tool - RPI/MAC
DUT - RPI

- Steps to reproduce :

DUT side : 

1. sudo rm -rf /tmp/chip_*
2. sudo ./chip-all-clusters-app --wifi

TH side : 

1. ./chip-tool pairing ble-wifi 1 zigbeehome matter123 20202021 3840 (on TH1)
2. ./chip-tool pairing open-commissioning-window 1 1 100 1000 3840 / ./chip-tool administratorcommissioning open-basic-commissioning-window 500 1 0 --timedInteractionTimeoutMs 1000 (on TH1)
3. ./chip-tool pairing manualcode 2 35484132896 / ./chip-tool pairing onnetwork 2 20202021 (on TH2)
4. ./chip-tool pairing open-commissioning-window 3 1 100 1000 3840 / ./chip-tool administratorcommissioning open-basic-commissioning-window 500 3 0 --timedInteractionTimeoutMs 1000(on TH3)
5. ./chip-tool pairing manualcode 3 35484132896 / ./chip-tool pairing onnetwork 2 20202021  (on TH3)
6. ./chip-tool operationalcredentials read fabrics 2 0 --fabric-filtered 0
7. ./chip-tool operationalcredentials remove-fabric 2 2 0

- After step 7 DUT gets aborted automatically

Screenshot : 
![image](https://user-images.githubusercontent.com/90545664/162147515-9e24c5d7-fb34-4728-a957-7d5f393c0caf.png)

PFA logs : 
[remove-fabric_cntrl.txt](https://github.com/project-chip/connectedhomeip/files/8440725/remove-fabric_cntrl.txt)
[remove-fabric_dut.txt](https://github.com/project-chip/connectedhomeip/files/8440726/remove-fabric_dut.txt)

**Comment by bzbarsky-apple** [comment]
Duplicate of #16748
